### ISSUE TITLE
View general summary of a submission

### ISSUE THREAD

**Issue by aaclan-ebi** [ebi-ait/dcp-ingest-central#42]

**Comment by clairerye** [comment]
That's great Alegria, thank you!

**Comment by ofanobilbao** [comment]
@gabsie I wonder if this story also would be useful in the context of error identifying earlier in the process. Thoughts?

**Comment by gabsie** [comment]
yes this could be linked to the ingest graph validator ticket, when displaying errors.
